docs/2_3_common_states.adoc

```diff
@@ -390,9 +390,11 @@ This function should always be called after calling function <<fmi3UpdateDiscret
 
 Function <<fmi3GetDirectionalDerivative>>::
 See <<fmi3GetDirectionalDerivative>>.
+This call is not allowed in Scheduled Execution.
```

Comment:
@andreas-junghanns @IZacharias I'm not sure if this should be forbidden. On the other hand I don't see why one would use it in SE. Maybe this is enough reason to forbid it...?